packages/core/src/utils-hoist/time.ts

```diff
@@ -116,18 +111,24 @@ export const browserPerformanceTimeOrigin = ((): number | undefined => {
   if (timeOriginIsReliable || navigationStartIsReliable) {
     // Use the more reliable time origin
     if (timeOriginDelta <= navigationStartDelta) {
-      // eslint-disable-next-line deprecation/deprecation
-      _browserPerformanceTimeOriginMode = 'timeOrigin';
-      return performance.timeOrigin;
+      return [performance.timeOrigin, 'timeOrigin'];
     } else {
-      // eslint-disable-next-line deprecation/deprecation
-      _browserPerformanceTimeOriginMode = 'navigationStart';
-      return navigationStart;
+      return [navigationStart, 'navigationStart'];
     }
   }
 
   // Either both timeOrigin and navigationStart are skewed or neither is available, fallback to Date.
-  // eslint-disable-next-line deprecation/deprecation
-  _browserPerformanceTimeOriginMode = 'dateNow';
-  return dateNow;
-})();
+  return [dateNow, 'dateNow'];
+}
+
+/**
+ * The number of milliseconds since the UNIX epoch. This value is only usable in a browser, and only when the
+ * performance API is available.
+ */
+export function browserPerformanceTimeOrigin(): number | undefined {
```

Comment:
Hmm this is a breaking change though, I guess? 🤔

Reply:
When I started this PR this was all in utils... I thought we don't consider anything in utils as part of our public API?